odk1-src/aggregate-deployment-planning.rst

```diff
@@ -85,23 +79,23 @@ Ongoing support
 Infrastructure architecture
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In these docs we explain how to deploy a monolithic server with everything Aggregate needs to run (Tomcat, PostgreSQL, SSL support), but there's a multitude of different scenarios that you might want to consider:
+In these docs we explain how to deploy a monolithic server with everything Aggregate needs to run (Tomcat, PostgreSQL, SSL support), but there are a multitude of scenarios that you might want to consider:
```

Comment:
I'm not a native English speaker, but I'm curious: is it not "there is a multitude of scenarios" (one multitude) as in "there is a pack of cigarettes" (one pack)?